src/serializable/pvm/disableSubnetValidatorTx.ts

```diff
@@ -0,0 +1,48 @@
+import { concatBytes } from '@noble/hashes/utils';
+import { pack, unpack } from '../../utils/struct';
+import { BaseTx } from '../avax/baseTx';
+import { Codec } from '../codec/codec';
+import { serializable, type Serializable } from '../common/types';
+import { TypeSymbols } from '../constants';
+import { Id } from '../fxs/common';
+import { PVMTx } from './abstractTx';
+import type { Input } from '../fxs/secp256k1';
+
+@serializable()
+export class DisableSubnetValidatorTx extends PVMTx {
+  _type = TypeSymbols.DisableSubnetValidatorTx;
+
+  constructor(
+    public readonly baseTx: BaseTx,
+    public readonly validationId: Id,
+    public readonly disableAuth: Serializable,
```

Comment:
Just following what other transaction types that take a auth do.

I think we could type this as `Input` but it will require us to cast the type on line 33 in the `fromBytes` method when passing to `new DisabledSubnetValidatorTx` since we need the `Codec` deserializier. Using `Input` as the deserializer doesn't work and will cause tests to fail.

So my thinking was to just keep it in line with the other transaction types.

Reply:
Just following what other transaction types that take a auth do.

I think we could type this as `Input` but it will require us to cast the type on line 33 in the `fromBytes` method when passing to `new DisabledSubnetValidatorTx` since we need the `Codec` deserializier. Using `Input` as the deserializer doesn't work and will cause tests to fail.

So my thinking was to just keep it in line with the other transaction types.